pkg/daemon/ceph/osd/volume.go

```diff
@@ -110,7 +110,7 @@ func (a *OsdAgent) configureCVDevices(context *clusterd.Context, devices *Device
 			osds = append(osds, lvmOsds...)
 
 			// List THE existing OSD configured with ceph-volume raw mode
-			if a.cluster.CephVersion.IsAtLeast(cephver.CephVersion{Major: 14, Minor: 2, Extra: 7}) && !lvBackedPV {
+			if a.cluster.CephVersion.IsAtLeast(cephver.CephVersion{Major: 14, Minor: 2, Extra: 8}) && !lvBackedPV {
```

Comment:
While we're making this change, how about defining the version in a single var so that we don't have these versions to change in many places?